### DISCUSSION TITLE
Owner Maintenance im "Admin Tools" Bereich

### DISCUSSION THREAD

**Discussion by ChristianGK-GC** [2Abendsegler/GClh#2181] [Ideas]
Hallo zusammen,

mir fehlt oft der Eintrag Owner Maintenance im "Admin Bereich" eines Caches.
Diesen nutze ich sehr oft im Vergleich zu Archive oder Disable.
Ich würde den Eintrag gern hinzufügen.
Die Reihenfolge ist mir egal, ob der Eintrag am Ende, Mitte oder Anfang steht.

Beispiel:
![grafik](https://user-images.githubusercontent.com/24353480/188642637-0e247b3a-06d5-48e5-9e01-2d01d6dc5190.png)

Gruß
Christian

**Comment by capoaira** [comment]
Ich wollte gerade schreiben, dass das der Standard Logtype ist, aber beim Testen ist mir aufgefallen, dass das nun endlich abgeschafft wurde. Von daher finde ich die Idee gut und sinnvoll.👍🏻

**Reply by capoaira** [reply]
Ich würde es unten anfügen, da so die bisherigen Links an selber Stelle bleiben.

**Reply by ChristianGK-GC** [reply]
Mir gefällt es!

![grafik](https://user-images.githubusercontent.com/24353480/188648740-f604ce08-0fb1-4026-a094-ee28e29d7dc4.png)

**Comment by ChristianGK-GC** [comment]
Ich hab mal eine Version gebaut.
https://github.com/ChristianGK-GC/GClh/blob/collector/gc_little_helper_II.user.js

**Reply by capoaira** [reply]
Sieht gut aus 👍🏻, kannst gerne ein PR machen

**Comment by ChristianGK-GC** [comment]
Wie kann das hier zu gemacht werden?

**Reply by capoaira** [reply]
Soweit ich weiß, geht das nicht
